app/assets/stylesheets/_summary-card.scss

```diff
@@ -0,0 +1,29 @@
+.summary-card {
+  margin-bottom: 20px;
+  padding: 15px;
+  width: 100%;
+  @include box-shadow(0 1px 4px rgba(0,0,0,.3));
+
+  .sc-row {
```

Comment:
Yes, I think that's a great idea. I will look into it and make the appropriate changes.